package.json

```diff
@@ -27,17 +27,14 @@
   ],
   "scripts": {
     "build": "kcd-scripts build --out-dir .",
-    "contributors:add": "all-contributors add",
-    "contributors:generate": "all-contributors generate",
     "lint": "kcd-scripts lint",
     "validate": "kcd-scripts validate"
   },
   "dependencies": {
-    "@babel/runtime": "^7.9.2"
+    "@babel/runtime": "^7.12.1"
   },
   "devDependencies": {
-    "all-contributors-cli": "^6.14.0",
-    "kcd-scripts": "^5.6.1"
+    "kcd-scripts": "^6.6.0"
```

Comment:
What are the breaking changes here? Is it safe to merge or does it depend on the other PR?

Reply:
It's safe to merge 🙂